css/types/length.json

```
@@ -290,8 +290,8 @@
               "webview_android": "mirror"
             },
             "status": {
-              "experimental": true,
-              "standard_track": false,
+              "experimental": false,
```

We do not mark "experimental" based upon spec status, but rather based upon browser support.

```suggestion
              "experimental": true,
```